test/inline_partial_require_test.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Sorry to bug you on this, but please use two spaces instead of four to match the rest of the code base.